smart-contracts/test/BridgeProxy/CCTPBridgeProxy.ethereum.ts

```diff
@@ -21,7 +21,9 @@ const recipient = '0x246A13358Fb27523642D86367a51C2aEB137Ac6C'
 
 const amount = ethers.parseUnits('0.1', 6)
 
-describe('CCTPBridgeProxy', function () {
+const relayPool = '0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266'
+
+describe.skip('CCTPBridgeProxy', function () {
```

Comment:
should not be skipped ?

Reply:
Yes because we need to change that test cimpletely! so that it tests the library rather than our contracts!